Controller/MediaAdminController.php

```diff
@@ -42,22 +41,20 @@ public function createAction(Request $request = null)
     /**
      * {@inheritdoc}
      */
-    public function render($view, array $parameters = array(), Response $response = null, Request $request = null)
+    public function render($view, array $parameters = array(), Response $response = null)
```

Comment:
@core23 please review again… can we at least agree that you got your example backwards in https://github.com/sonata-project/SonataMediaBundle/pull/1196#discussion_r100724466 ?